sdk/notificationhubs/notification-hubs/src/models/options.ts

```diff
@@ -2,22 +2,68 @@
 // Licensed under the MIT license.
 
 import { CommonClientOptions, OperationOptions } from "@azure/core-client";
+import { BrowserPushChannel } from "./installation.js";
+import { objectHasProperty } from "@azure/core-util";
 
 /**
  * Describes the options that can be provided while creating the NotificationHubsClientContext.
  */
 export interface NotificationHubsClientOptions extends CommonClientOptions {}
 
 /**
- * Represents the send operation options that can be set.
+ * Options for sending notifications for both tag based send and broadcast scheduled send.
  */
-export interface SendOperationOptions extends OperationOptions {
+export interface ScheduleNotificationOptions extends OperationOptions {
+  /**
+   * The tags used to target the device for push notifications in either an array as "an || condition" or tag expression.
+   */
+  tags?: string | string[];
```

Comment:
would it be worth to add a ref doc comment about broadcasting when tags are not specified or empty?

Reply:
Added docs to indicate broadcast if not set